README.md

```diff
@@ -13,3 +13,26 @@ This guide provides a uniform way to structure your RiotJS code. Making it
 * easier to cache and serve bundles of code separately.
 
 This guide is inspired by [AngularJS Style Guide](https://github.com/johnpapa/angular-styleguide) by John Papa.
+
+## Table of Contents
+
+1. [Component / Tag names](#component-tag-names)
+
+## Component / Tag names
+
+Component or tag names need to be:
+
+* **Spec compliant**: [include a hyphen](https://www.w3.org/TR/custom-elements/#concepts).
+* **Meaningful**: not overspecific, not overly abstract.
+* **Short**: 2 or 3 words.
+* **Pronouncable**: we want to be able talk about them.
+* **`app` namespaced**: using a standard namespace allows easy reuse in other projects
+
+*Why?*: Your `tag`s are custom elements in the document. As such, they need to adhere the spec.
+
+```markup
+<!-- recommended -->
+<app-header />
+<app-user-list />
+<app-ui-slider />
```

Comment:
A slider is always a UI component and always part of an app in some way. Meaning `ui` and `app` are not really  meaningful, are making the name longer and less pronounceable.

So in this case, this could just be `<slider>` except that that breaks the spec compliant rule. Therefore I would try to add what kind of slider it is. So for instance `<range-slider>` or `<hue-slider>` etc.